cmake/CMakeLists.txt

```diff
@@ -1135,7 +1135,7 @@ endif()
 
 if(PKG_USER-INTEL)
   include(CheckIncludeFile)
-  check_include_file(immintrin.h FOUND_IMMINTRIN)
+  check_include_file_cxx(immintrin.h FOUND_IMMINTRIN)
```

Comment:
You will need to update the include above as well.

Reply:
not really needed here, since this is already included globally.